build_iso.sh

```diff
@@ -24,6 +24,10 @@ if [ ! -f "$wd"/etc_fai/grub.cfg ]; then
   cp "$wd"/etc_fai/grub_base.cfg "$wd"/etc_fai/grub.cfg
 fi
 
```

Comment:
build_tmp must be created if it does not exists

Reply:
I will use .gitkeep files to have the folder created when cloning the repo.